release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -1169,6 +1169,7 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * Due to a race condition, the {rh-openstack-first} cloud provider might not start properly. Consequently, LoadBalancer services might never get an `EXTERNAL-IP` set. As a temporary workaround, you can restart the kube-controller-manager pods using the procedure described in link:https://bugzilla.redhat.com/show_bug.cgi?id=2004542[*BZ#2004542*].
 
+* The SR-IOV network configuration daemon pod will cordon the node and mark it as unschedulable. It will use the add or delete `SirovNetworkNodePolicy` custom resource (CR) before waiting for the `syncStatus` of the CR to change to `Succeeded`. As a temporary workaround, before adding or deleting a `SriovNetworkNodePolicy` CR, make sure the `syncStatus` of the `SriovNetworkNodeState` CRs is in the `Succeeded` state. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002508[*BZ#2002508*])
```

Comment:
@yingwang-0320 could you verify this once more. I have made a few edits based on some peer review feedback and I want to make sure everything still looks good. Specifically, the "add or delete `SirovNetworkNodePolicy` custom resource (CR)" part. Thank you!

Reply:
@opayne1 it looks good to me

Reply:
/lgtm